pkg/bundleyaml/bundleyaml.go

```diff
@@ -0,0 +1,39 @@
+package bundleyaml
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	fleetYaml         = "fleet.yaml"
+	fallbackFleetYaml = "fleet.yml"
+)
+
+func FoundFleetYamlInDirectory(baseDir string) bool {
+	if _, err := os.Stat(GetFleetYamlPath(baseDir, false)); err != nil {
+		if _, err := os.Stat(GetFleetYamlPath(baseDir, true)); err != nil {
+			return false
+		}
+	}
+	return true
+}
+
+func GetFleetYamlPath(baseDir string, useFallbackFileExtension bool) string {
+	if useFallbackFileExtension {
+		return filepath.Join(baseDir, fallbackFleetYaml)
+	}
+	return filepath.Join(baseDir, fleetYaml)
+}
+
+func IsFleetYaml(fileName string) bool {
+	if fileName == fleetYaml || fileName == fallbackFleetYaml {
+		return true
+	}
+	return false
+}
+
+func IsFleetYamlSuffix(filePath string) bool {
+	return strings.HasSuffix(filePath, "/"+fleetYaml) || strings.HasSuffix(filePath, "/"+fallbackFleetYaml)
```

Comment:
I barely know anything about Fleet, but is it ever run on Windows? If yes, should this path handling be platform-agnostic?

Reply:
Looks like `path/filepath` has this:
```func HasPrefix(p, prefix string) bool```

Reply:
You're absolutely right that we should likely not be wrangling file paths as strings. Path separators should be handled by a library. However, I'd prefer for this PR to retain existing logic since there are _multiple_ places where this would need to be change to keep logic consistent. Thus, I've opened this issue: https://github.com/rancher/fleet/issues/599

![Screen Shot 2021-11-08 at 1 46 53 PM](https://user-images.githubusercontent.com/39320683/140799806-41bac747-cff6-421e-b14c-b2a793ef5e86.png)
s